dagconfig/params.go

```diff
@@ -185,6 +186,16 @@ type Params struct {
 	HDCoinType uint32
 }
 
+// NormalizeRPCServerAddress returns addr with the current network default
+// port appended if there is not already a port specified.
+func (p *Params) NormalizeRPCServerAddress(addr string) string {
+	_, _, err := net.SplitHostPort(addr)
```

Comment:
Add a comment about blabla, here and any other way where we might use SplitHostPort in a similar way